### PR TITLE
fix: firefox does not connect correctly close jira-42

### PR DESCRIPTION
**Pull request by Jack-Works**
closes jira-42

### CODE REVIEW

File: packages/maskbook/src/extension/background-script/EthereumServices/send.ts [hunk 1]
```diff
@@ -22,8 +22,8 @@ export async function INTERNAL_send(
     callback: (error: Error | null, response?: JsonRpcResponse) => void,
 ) {
     if (process.env.NODE_ENV === 'development') {
-        console.table(payload)
-        console.log(new Error().stack)
+        // console.table(payload)
+        // console.log(new Error().stack)
```
Comment (guanbinrui) [inline packages/maskbook/src/extension/background-script/EthereumServices/send.ts:26]:
```suggestion
        console.table(payload)
        console.log(new Error().stack)
```

Comment (Jack-Works) [reply packages/maskbook/src/extension/background-script/EthereumServices/send.ts:26]: it prints massive logs in the background

File: packages/maskbook/src/plugins/Wallet/settings.ts [hunk 2]
```diff
@@ -120,4 +120,6 @@ export const currentGasNowSettings = createGlobalSettings<GasNow | null>(
     (a: GasNow | null, b: GasNow | null) => isEqual(a, b),
 )
 
-connectGasNow()
+try {
+    connectGasNow()
+} catch {}
```
Comment (septs) [inline packages/maskbook/src/plugins/Wallet/settings.ts:125]: plz note why need ignore catch

Comment (Jack-Works) [reply packages/maskbook/src/plugins/Wallet/settings.ts:125]: It runs on the top level and it is the critical path of our program.

Comment (septs) [reply packages/maskbook/src/plugins/Wallet/settings.ts:125]:
```suggestion
} catch {
    // it runs on the top level and it is the critical path of our program.
}
```

Comment (Jack-Works) [reply packages/maskbook/src/plugins/Wallet/settings.ts:125]: Actually, the correct fix is to move it elsewhere. I'll draft an issue for it.

**Review by Tedko — APPROVED** [review]

**Comment by Jack-Works** [comment]
And this should be picked in to the next release

**Pull request merged**